src/fides/core/export_helpers.py

```diff
@@ -194,9 +223,17 @@ def format_data_subjects(
     ]
 
     formatted_data_subjects: Dict[FidesKey, Dict[str, str]] = {}
+    excluded_attributes = (
+        "name",
+        "rights_available",
+        "automated_decisions_or_profiling",
+    )
```

Comment:
So I think what I am seeing is there is an attribute (`system.privacy_declaration.data_subjects.tags`) that wasn't being included when exporting previously as we were being exclusive as to what was allowed in an export. When adding a `custom_field` it is now inclusive of everything in the base model judging the remainder to be custom fields. When adding tags in here as an `excluded_attribute` and into `generate_system_records()` it appears to lead to a bit of whack-a-mole with other attributes.

I think it may be a question of if we want to trace down all the attributes and maintain them inclusively or if we want 
 to treat custom fields as a separate defined object. You've been deep into this lately as well, any other ideas potentially?

Reply:
It turns out this had already been hit with `DataUse` and had the fields explicitly applied. I replicated the same style for `DataSubject` and everything was able to succeed as expected